Gemfile

```diff
@@ -5,20 +5,20 @@ gem "sinatra", "1.3.4"
 gem "rake", "~> 10.5"
 gem "rack", "~> 1.6"
 gem "rest-client", "1.8.0"
-gem "logging", "1.8.1"
-gem 'nokogiri', "1.5.5"
+gem "logging", "2.1.0"
+gem 'nokogiri', "1.6.7.2"
 gem 'whenever', require: false
-gem "slop", "3.4.5"
-gem "sidekiq", "2.13.0"
+gem "slop", "4.3.0"
+gem "sidekiq", "< 3"
```

Comment:
It does work - bundler will resolve things the Gemfile to any "valid" version - there is no guarantee that it will take the latest. If you want a specific version you need to tell it to use that specific version.

The latest 2.X from https://rubygems.org/gems/sidekiq/versions is `2.17.8` so you'll need to put that in the Gemfile.